axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/service/bankstatement/BankStatementService.java

```diff
@@ -294,18 +294,21 @@ public void checkAmountWithinBankStatement(
         deleteLines = true;
         break;
       }
+
       for (int i = 0; i < initialBankStatementLineAFB120.size(); i++) {
```

Comment:
Better to use a foreach loop for browse a list like for (BankStatementLineAFB120 bankStatementLineAFB120 : initialBankStatementLineAFB120) { instead of use a counter